device-types/HPE/OfficeConnect-1920-48G-PoE-370W.yml

```diff
@@ -0,0 +1,215 @@
+---
+manufacturer: HPE
+model: OfficeConnect 1920-48G-PoE-370W
+slug: hpe-officeconnect-1920-48g-poe-370w
+part_number: JG928A
+u_height: 1
+is_full_depth: false
+comments: '[OfficeConnect 1920 Switch Series - Specifications](https://support.hpe.com/hpesc/public/docDisplay?docId=c04478097&docLocale=en_US)'
+weight: 4.3
+weight_unit: kg
+power-ports:
+  - name: PS1
+    type: iec-60320-c14
+    maximum_draw: 492
+interfaces:
+  - name: '1'
+    type: 1000base-t
+    poe_mode: pse
+    poe_type: type1-ieee802.3af
```

Comment:
Looking at the spec sheet this switch is POE+ therefore the correct poe_type is `type2-ieee802.3at`